From e3acb11a2ba4b85aa7d6f204a3921cd4ce8576ad Mon Sep 17 00:00:00 2001
From: Claudiu Manoil <claudiu.manoil@nxp.com>
Date: Wed, 10 Apr 2019 19:05:29 +0300
Subject: [PATCH 693/706] mscc: ocelot: Add NXP copyright for modified files

Addresses legal issues for changes following changes
on ocleot.c  and ocelot.h:

604978a ("felix: fix null pointer on adding mdb entry")
e61c1fc ("felix: add general config register block")
fea3afb ("ocelot: fdb: add new MAC entry for CPU port on PGID_CPU")

Signed-off-by: Claudiu Manoil <claudiu.manoil@nxp.com>
(cherry picked from commit 9a298d4e59dc769e5017637ca7d2d3cb70c1c468)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/mscc/ocelot.c | 1 +
 drivers/net/ethernet/mscc/ocelot.h | 1 +
 2 files changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/mscc/ocelot.c b/drivers/net/ethernet/mscc/ocelot.c
index e90082100d63..b9ac38cb927d 100644
--- a/drivers/net/ethernet/mscc/ocelot.c
+++ b/drivers/net/ethernet/mscc/ocelot.c
@@ -3,6 +3,7 @@
  * Microsemi Ocelot Switch driver
  *
  * Copyright (c) 2017 Microsemi Corporation
+ * Copyright 2019 NXP
  */
 #include <linux/etherdevice.h>
 #include <linux/ethtool.h>
diff --git a/drivers/net/ethernet/mscc/ocelot.h b/drivers/net/ethernet/mscc/ocelot.h
index 18ba196a2ad6..a52febe5d708 100644
--- a/drivers/net/ethernet/mscc/ocelot.h
+++ b/drivers/net/ethernet/mscc/ocelot.h
@@ -3,6 +3,7 @@
  * Microsemi Ocelot Switch driver
  *
  * Copyright (c) 2017 Microsemi Corporation
+ * Copyright 2019 NXP
  */
 
 #ifndef _MSCC_OCELOT_H_
-- 
2.17.1

